### PR TITLE
Profile layout improvements

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- load the new profile
- open the list of charms
- check that a list of tags is displayed for each charm
- open the model list
- check that the second column displays the owner

**Comment by huwshimi** [comment]
[Profile cleanup](https://trello.com/c/l0rlXIVF/168-profile-cleanup)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8390/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8391/

File: jujugui/static/gui/src/app/components/profile/charm-list/charm-list.js [hunk 1]
```diff
@@ -83,12 +83,58 @@ class ProfileCharmList extends React.Component {
...
+      </ul>);
+  }
+
   render() {
     let content;
     if (this.state.loading) {
       content = (<Spinner />);
     } else {
       const rows = this.state.data.map(charm => {
+        console.log(charm);
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/profile/charm-list/charm-list.js]: oops

**Comment by hatched** [comment]
Thanks @huwshimi :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8392/

**Comment by hatched** [comment]
Spurious CI failure

**Pull request merged**